fido-uaf-core/src/main/java/com/nexenio/fido/uaf/core/msg/AuthenticationRequest.java

```diff
@@ -20,8 +20,26 @@
 
 @Data
 public class AuthenticationRequest {
-    private OperationHeader header;
+
+    /**
+     * Must be "AUTHENTICATION"
+     */
+    private OperationHeader operationHeader;
```

Comment:
You must NOT change the names in the `msg` package, since this changes the serialization as well, which is part of the spec. The name `header` needs to stay as it is.
Please see https://fidoalliance.org/specs/fido-uaf-v1.1-id-20170202/fido-uaf-protocol-v1.1-id-20170202.html#authenticationrequest-dictionary